### PR TITLE
Add a class to store the stat var entries used to compute page config

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Could you add a meaningful title please :)

File: server/lib/nl_variable.py [hunk 1]
```diff
@@ -13,11 +13,34 @@
...
 
 
+class EntryType(Enum):
+  SINGLE = 1
+  PEER = 2
+
+
+@dataclass
+class Entry:
+  # Only one of single and peers should be set.
```
Comment (pradh) [inline server/lib/nl_variable.py]: s/single/source/?

File: server/lib/nl_variable.py [hunk 2]
```diff
@@ -13,11 +13,36 @@
...
+
+
+@dataclass
+class SimpleEntry(Entry):
+  dcid: str
+
+
+@dataclass
+class PeerEntry(Entry):
+  # sources contains the sv/svg from the model. They are used to deduce peer
```
Comment (pradh) [inline server/lib/nl_variable.py:36]: sv/svg/topic ?

**Review by pradh — APPROVED** [review]
Thanks!

File: server/lib/nl_variable.py (hunk 1 above)
Comment (shifucun) [reply server/lib/nl_variable.py]: I will update in next PR

**Pull request merged**